src/pages/Admin/Venue/VenueWizard/Create/VenueWizardCreate.tsx

```diff
@@ -39,11 +37,7 @@ const VenueWizardCreate: React.FC<VenueWizardCreateProps> = ({
     );
   }
 
-  return (
-    <AdminRestricted>
-      <Details previous={previous} dispatch={dispatch} data={state} />
-    </AdminRestricted>
-  );
+  return <Details previous={previous} dispatch={dispatch} data={state} />;
```

Comment:
Why remove `AdminRestricted`?
Only people with admin rights should be able to create venues.

Reply:
In order to update styles, a simple `VenueWizardCreate.scss` can have

```scss

.VenueWizardCreate {
  .AdminRestricted {
    // scrollbar fix
  }
}

```

This is the flexibility provided by diligently setting up block classes on all root elements of all components.